### DISCUSSION TITLE
Virtual Folder create

### DISCUSSION THREAD

**Discussion by Pekka893** [requarks/wiki#2790] [Help / Questions] [answered]
**Question**
Hello how i can create an virtual folder?

**Host Info (please complete the following information):**
OS: Ubuntu 20.04
Wiki.js version: 2.5.170
Database engine: mysql 5.7.32

**Comment by NGPixel** [comment, marked as answer]
You don't. Create the page at the path you want, folders are "created" automatically. See https://docs.requarks.io/guide/structure
